blog: add pitfalls of modern platform engineering

ericgregory opened the pull request:
Adds "Pitfalls of modern platform engineering" blog, to be published 12/12.

ericgregory committed:
blog: add pitfalls of modern platform engineering

Signed-off-by: Eric Gregory <eric@cosmonic.com>

CazJT committed:
Update index.mdx

Tiny edits and the addition of a collapsable section for the transcript. @ericgregory adding this to see what it looks like and to take your view. This would not affect SEO.

Signed-off-by: CazJT <113366164+CazJT@users.noreply.github.com>

CazJT reviewed (commented):
Addition of a collapsible section for the transcript - for discussion with @ericgregory

CazJT committed:
Merge branch 'main' into pitfalls

ericgregory committed:
Add video

Signed-off-by: Eric Gregory <eric@cosmonic.com>

ericgregory committed:
Adjust video

Signed-off-by: Eric Gregory <eric@cosmonic.com>

ericgregory committed:
Adjust video again

Signed-off-by: Eric Gregory <eric@cosmonic.com>

CazJT committed:
Update index.mdx

Removed collapse to enable signposting. Thanks @ericgregory final check and merge! TY

Signed-off-by: CazJT <113366164+CazJT@users.noreply.github.com>

CazJT reviewed (commented):
Final edits in
